duffel_api/models/__init__.py

```diff
@@ -1,13 +1,12 @@
 from .aircraft import Aircraft
-from .airport import Airport, City
 from .airline import Airline
+from .airport import Airport, City, Place, Refund
```

Comment:
Definitely want to change this - sorry I haven't got round to a better solution yet! I'll add an issue for it and assign it to myself so I don't forget!